examples/addsvc/client/main.go

```diff
@@ -49,6 +58,25 @@ func main() {
 	logger = log.NewContext(logger).With("caller", log.DefaultCaller)
 	logger = log.NewContext(logger).With("transport", *transport)
 
+	// Set up OpenTracing
+	var tracer opentracing.Tracer
+	{
+		if len(*appdashHostport) > 0 {
+			tracer = appdashot.NewTracer(appdash.NewRemoteCollector(*appdashHostport))
+		}
+		if len(*lightstepAccessToken) > 0 {
+			if tracer != nil {
+				panic("Attempted to configure multiple OpenTracing implementations")
+			}
+			tracer = lightstep.NewTracer(lightstep.Options{
+				AccessToken: *lightstepAccessToken,
+			})
+		}
+		if tracer == nil {
+			tracer = opentracing.GlobalTracer() // the noop tracer
+		}
+	}
```

Comment:
@peterbourgon if we intend to keep more than one of these bindings, I'll definitely clean this up and change the flag names, etc... I just included both LightStep and Appdash to demonstrate how easy it is to switch them for this RFC. Preferences/thoughts on this?